### ISSUE TITLE
The hover state of button component is not correct

### ISSUE THREAD

**Issue by fcoveram** [cc-archive/vocabulary-legacy#372]
## Describe the bug
The button has a different background color and it has an underline style during the hover state. In the default state, the icon has an incorrect color.

## Expected behavior
1. It should not have an underline style
2. The background color should be the interaction color defined in [the Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A72&viewport=45%2C274%2C0.15307241678237915).
3. The icon should be white with 50% opacity in default and hover state.

## Screenshots
<img width="509" alt="Screen Shot 2020-05-14 at 12 09 24" src="https://user-images.githubusercontent.com/895819/81958359-e0cb0780-95db-11ea-9d5a-7e18d9d48391.png">

**Comment by bigdevlarry** [comment]
@brenoferreira I see you are transferring the issue but I'm curious as to know why . Although I just started contributing yesterday so I have a little understanding .

**Comment by bigdevlarry** [comment]
@panchovm has the issue been assigned yet ? Would like to work on it 😊

**Comment by dhruvkb** [comment]
@LarrySul you can work on it but only after you've resolved the other issues assigned to you.

**Comment by bigdevlarry** [comment]
@panchovm could you help with the value of the defined background color ? @dhruvkb noted !

**Comment by fcoveram** [comment]
Here is [the Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A72&viewport=-490%2C-31%2C0.7026233673095703). Page **Color**.
1. Background color, default state: `ForestGreen`
2. Background color, hover state: `BrighterForestGreen`

**Comment by bigdevlarry** [comment]
@panchovm alright ...

**Comment by Dishebh** [comment]
@dhruvkb @LarrySul I would like to work on the issue.

**Comment by bigdevlarry** [comment]
@Dishebh please go ahead with it ...

**Comment by Dishebh** [comment]
@panchovm  @LarrySul can you please tell how do I replicate the issue? All the buttons I see in the repo do not have any style underline on hover. I also don't see the button (given in PR description) in the project website.

**Comment by bigdevlarry** [comment]
@Dishebh that's more reason I wasn't able to make a PR to close the issue so I thought I probably wasn't doing something right, so I wanted to let someone else try it out ... Everything seems okay to me tho . I guess someone else already fixed this from another PR that was merged

**Comment by Dishebh** [comment]
@panchovm I guess the issue has been resolved.

**Comment by fcoveram** [comment]
Let me tag @dhruvkb since he is more aware of all ticket's progress.

**Comment by dhruvkb** [comment]
@Dishebh this issue arises because of a feature that allows anchor links to be styled as buttons. None of the Button stories on Vocabulary demonstrate this feature (which is a shortcoming of the docs that you should ideally address when solving this issue).

To see the issue, you can see the underline by hovering on the "Go to image's website" button on [CC Search](https://search.creativecommons.org/photos/1212275a-c4fe-44e6-ae86-419e6f3f0c90).

To reproduce this issue, style an anchor tag like so:
```html
<a class="button is-success" href="#">
  I'm pretending to be a button
</a>
```

**Comment by Dishebh** [comment]
@dhruvkb so should I make a separate section under buttons that have both buttons as buttons and anchor tags styled as buttons, and then correct the styling of the anchor buttons (as per this issue description)?  Because, as you mentioned, there are no anchor tags (currently in the storybook) styled as buttons having class `.is-success`. Please let me know if we are on the same page here. Thanks!

**Comment by dhruvkb** [comment]
@Dishebh you can just add the corresponding anchor-tag buttons in all the stories of the button-tag buttons but fix their styling so that they work exactly the same?

**Comment by Dishebh** [comment]
![Screenshot from 2020-05-29 15-11-05](https://user-images.githubusercontent.com/43489708/83245646-a908b600-a1be-11ea-84b8-98cfa2508817.png)

@dhruvkb something like this? (Both having the same styling)

**Comment by Dhruvi16** [comment]
I guess these issues are fixed.
